app/retail/views.py

```diff
@@ -245,9 +245,18 @@ def how_it_works(request, work_type):
     """Show How it Works / Funder page."""
     if work_type not in ['funder', 'contributor']:
         raise Http404
-
+    title = ''
+    desc = ''
+    if work_type == 'contributor':
+        title = 'How to Find & Complete Open Bounties | Gitcoin'
+        desc = 'Learn how to get paid for open bug bounties and get paid in crypto (ETH or any ERC-20 token)'
+    if work_type == 'funder':
+        title = 'How to Create & Fund Issues/Bounties | Gitcoin'
+        desc = 'Learn how to create open bug bounties and get freelance developers to complete your job/task.  '
     context = {
-        'active': f'how_it_works_{work_type}',
+        'active': 'how_it_works_{work_type}',
```

Comment:
We need to place the `f` back in front of this string.  The above will result in:  `'how_it_works_{work_type}'` versus `'how_it_works_contributor'` and `'how_it_works_funder'`